### ISSUE TITLE
Failing to save

### ISSUE THREAD

**Issue by maisonaustyn** [Mezeman1/idle-ant-farm#24]
Title

**Comment by Mezeman1** [comment]
Failing to save? When?

**Comment by maisonaustyn** [comment]
Anytime it tries to auto save or even if I manually save it. So whenever I tab out and back in. It's loading to the last chance it had to save.

![Screenshot 2024-09-15 105623](https://github.com/user-attachments/assets/09ba32f2-6568-4173-95d0-81268fe0cf29)

**Comment by maisonaustyn** [comment]
I deleted my cookies to play as guest and it's saving there. So the update might have killed my save somehow

eyJpdiI6Wzc4LDE2MiwxNDIsNzYsMTk4LDM0LDIzOSwxMzksMjQ3LDYyLDQ1LDEwM10sImRhdGEiOiI3SHo1cTNJS3VaekhIZGJ3VGwxRlRzdU55NTRjSVVZcE93ekp0NTB4Tzg1RUc5VGo0SmpuKzNjVXIxRDkzRDBxK3BvcXF5MWtxUWdROFpHRlpaQ1BKRi9Iemdka1h6UWNlYW90aUxHa1NHMSt2ODQwc0o1dlZIaSttWFdxQjFFZkkzN3oxczNKeTNFdWRsbVlDdGRNRmpORFcySWpsRHo2SVgvSHdCeWJCRnNhRmZTN1Q0RnV1QkxvY0ZIM3JiVXJmcVlYckhqVFEzTlM5NFhNVGZBZFU3QVdqd1h5T2p1ZTBSRzVJdXRkcFEwMXdia09JL0s0eExPc2w4dHZkcFE4SUpzaEJ6dGxHbnpXVjBFMnFjblNNeFRLcVdnTFljUnRLQXF5WkJETDZUdE0zc0FubEdtNGw4U25Lcnl0YnE2cWlwWmpyMHY3N0EwU2l4MlVNVEFoUWNMekdkdGNUWWx2OVQ3RklwdHNEamRsYUl1YlZZYzZFT3RKcEZOcHFUdGJROGhlakhnRFloRzNmdXNOYTRMNWlkcUtpLzJlM1Zvci9aZExkdTI5aWdBdi9kL1hzdUlvYzNqb2xJek4vNVRNTXV0d1NZTVR6bitYNmpRdWNWREUzbm9HaHFvU05ka0kzRU93bkV1UTdnWDZUOVZLMUpKVVhDdlR4V3lNVHhhVE5xdnl5TmFGYzF1NjRuQ014SHMweURjNGF5cDVWTUd4OFk2c1FSTkQxMWZvVnJ4b0RqeHNMZGFKeEtUMDh3OWhOSGhqK1hJZ3dHN05aUzlLNGpsYVByakQ5MGhVKzVlclYvM3hqc2UrWW91clNZUjk0VmVJbzJlNGd6VzFMQ1hBeXVHaXI0L3pweGtJOUNmeWQ0dkxmL0VQTWV6R2RDcHpmeW1PdTBVeHllNXZnWnVub0czVGs0Vm9ZNXNFeFI5V2wrMEc2cS9yZUFYMGdTZjVpVTZsdnRTbjFWY1VTQU94UXk4elhSN3RVUGpVT1hBVjJYdE5tbDlKd3U0Vll6VGlJVDd4YlVPS2tlQmgxSFYrZlFjVm1qbVFqVHZGMkZvclM0RVA0ZmFpY015QlEzRk1aMkgvOU10ZXVjVjJvSXMvVlJyY2pyNU9aZnhuTEhFLzNKL1RnNjhjeWgvdnVFZ2dnRG9CNG16RTVzaW92SDI0MlIrbTFwS2QyWnloRk1vS3laaGo2cnk4dVg3QmZhclZtaW0rSVFndTd1ZG02QlhFY0ZDS29OU0xNdjBhbzJyMEJPRndqYkdXMFFPT21jb3Nkbm1HQTlNUkpYQmZRSW0ybGFQbnFveHlwN0xrWmdYT2pRODd0UWNyRnVXaVJKTlVUZitiSjliQXd6dzU3dWs4VG9sSXZyUHBOWE5hL1FVZjBXTXV4bktpSndINkpESUZOb1hXZ1dvVldSWHl0V29jejZ2bnJnZngrMDdFWmJ2Ymt3NXJVQUQ0V1g5cC9oTlpnY0xzWlVkMkhvOVVITTF1b01nT3JjdjNyVFZUQUl5VEpJM2ZKd1lOSmJEVU9HNmhmUmo3am9OV29zU1lCTitodTlkcGxsUU9taVZ5b21PYnl2VVpqVGtWNVUrRHpWd1pHS1hSVnUvTlFSUjRoeWRxR3JrUXppOURWSDRyVnErVG1QR0xubWd5bkNUVTBLeFVUcVhodGlodEl5MFAvdUZjb2gvYm9ocTdYN210Rk9Qa09TdkZYYU1aQWk2ZHFNSXN0a3UvUWlmaW9qN0txeXhjK0llQUlMbnZRckIxb3hyQ09PMHdOd291UXo1SHZORFp0U0xjc0ZLZ3R1UnZaMUl0Z3pCOVh3Q1IvZGVLd2xiS1dWU1FmMDJ3a1ZvLzJLR3pJR0RFV3Q3cVA0UXM3N0VETG9uRDUwT3BGZlljYklBYktFM1B2czBWT3AwTzZlOVRDNFRtZkxZb0pUVVRSaWpsdzc4Q0IwTm1aa3UwdUJYNFJBeXRENGVPdUwwOXFPM2F4T3Rzcit5K3BjbnFnc04wYlE1TlhQRXl6SWFhYWllaHFCd21rZVBXanJXb2Q5TmxpNXNxUHBFOTNhMjFkaks4L3B6SUxkWlR2YS9sakpjbFZldHFjRXdKRFhEVGluNDRuajVYQ3JrZmFtc2l4c3VHNkZHN3Qvdmk0azVLMXY5VjdPT0J6YTM5VFFZTkU2MHUrZG9LZS9zbGN5VEQyNytnZEUrdHBad2pWY1VHTXdlbSswMTNtUFB3TllXaGRoMGdJbWVqVFl4NVdSQkZzcEJCdElyYnFTQW5FeG9jcDdYR09MZndrVFNMMUxhS0tvbzY3eSt3Tm9SRUc3TlJleksvemlCNkxEQ3lDSVZ2YWF2djdRNU43S0xNZkZPRzJINmdac045RjM0amxuOHZSSUFvNmJMVXFQOUFhRVptUHdyc1BaVk9zbDNUZ0J6UkZ3V21rajhtc3FXdnlROFFKYWIxazFneFliSW1IcmY5UHQ4Y3J5eFZsYmtFb3c4VUlZb0tRTGgyQSt0bW9jZ1hTSWhGZUN6M0d3dThMS0pPRHZkem81NWF1SUFLTVZCUjdhSVdObGN0ZXpGSzhnYm5wOWxROHBUVlR5WkR3L2VMd3JZQzVQZ2tLSVFIa1Z1bEx6RytwYWIrTnRqN2tQaFBpUlZUM1QyZWpZYjZKckZZM2x0dHJ5YStTZ3lZaWEyd0NyVWVnUjBOL3lmWEZ5R0xLYlhuNkZvRXF1bTlZZHUycFE5VGpwVGFUbFhuNFZRbFNCRzdGbWMrdGZWanFFRmJVQXlGcU4vUGpwZGZRTVlvK0NkTG5KWVZCVzVxV1hWbFdaMllGa1RkKytnOVc5eHdyWTdPaUE3N2Y4VlNMaXpYS0l4VXQrR05VSi9hS3l3TzAyLzJwdjlFbTd3N0lkSlh3YnZ1cUYvUWh0Q2RlNU8yMHRSMGdMWWw3Wjc3UGU3R3RLZ0tIK2tQQVZJS25YamsrQkFmd0NBNlFQVmx5WDJVdVorQzBNb0E2L1crQWJIUkpYNGFybVRLQktMSk4vdXpOSDVVcjBxMmdtamdnSHdmeU5YM0JOYXN1bDhRaXl4aUdYZHdZZS85YVIyU0VTQVZBUHJSZHZzWmVJSU1RSGtQem9xaW9KYnV0ODN2OUkzd043Q3JybjNVTm0zQ0hFMGd3YXhuVnJ1Y09lTzU3OUVwL3pqbnNDREZQeUZBaTFUdm1CNllPTGprdUdiTTlqZGVBQ0JHMDk3d0dLMVZ5enZaSXgvL2FrT0ZPUWpNKysyazEzMFV1VE9TcHhhVXJrcXRMYS9TdlR4ZVQ1amVmQmlVNEphM0o2V09xV2NDaUlXQlRDSDBkZk16REQxWldyaHVxNUJSTXJic3cxanQrUmpEdVVsdzIvTjJSUkZiWnp5QUMwZ0U3M3BSaEI5TGVnOU45WVZUR2dtT2xSVEEreHI5aUpabUZqOW8xQWVaSUFodEt4WmRGdlVaR0VLTlpaVXZRRURMY0pHTzFNVytyVW9oYmtaVE1kYzh6R3FBNTg1Z2kveVdhTDViaUpzSmlUMGhydUE0UklIMTg2YVRBdFM3T2ZScXBDZzhVL1NOZmVZaFpRQUt0UWpkWTM1VVMrYVZYU2pzU0RSR05GdEw3blRyLzNtc3MzVFU0WUR1Q1NDL0dlRGRGN0hKRVJiMWFtZE8ifQ==

Bugged save for reference. Not 100% certain if it's something I did or it was from the update

**Comment by Mezeman1** [comment]
You can export the save in settings, and import it with a new guest login.

**Comment by Mezeman1** [comment]
It's from clearing your cookies, it will lose reference to who you are

**Comment by maisonaustyn** [comment]
No lol. I cleared my cookies after this was already an issue to see if it's tied to my save specifically and it appears to be the case. I have an account made.

The exported save is from my account

I imported the save above to a guest account and it's not saving. So it's the save itself. If I start over. The game saves normally

Which is why I say I'm unsure where the problem originated from. Something I did in game or the update
